app/public-path.js

```diff
@@ -0,0 +1,6 @@
+// Allow overriding the webpack public path
```

Comment:
I think you just need to tell it that this is an expected global!

```suggestion
/* global __webpack_public_path__ */
// Allow overriding the webpack public path
```

Reply:
Sounds like eslint needs a "webpack" environment which understands the __webpack_*__ globals:

https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
This fixes the read-only thing:

```js
/* global __webpack_public_path__:true */
```

But it still complains about:

```
  1:11  error  '__webpack_public_path__' is defined but never used. Allowed unused vars must match /^_/  no-unused-vars
```

Despite the name matching `/^_/`!

Reply:
Figured it out! I need an `/* exported __webpack_public_path__ */` as well!

Reply:
Aha! Sorry for the bumsteer!